views/files/files-grid.hbs

```diff
@@ -26,43 +28,53 @@
 								{{#ifeq this.securityCheck.status "blocked"}}
 									<a href="#"
 									class="btn-file-danger"
-									title="{{$t "files.label.dangerousCode"}}">
+									title="{{$t "files.label.dangerousCode"}}"
```

Comment:
Do we need the title here when we have an aria-label as well?

Reply:
The Title is needed for the pop-up afaik:
![title](https://user-images.githubusercontent.com/45626086/91283859-72294880-e78b-11ea-8a53-88a47d389955.png)
Is the title sufficient? Should I remove the aria-label then?

Reply:
Yes, then I guess we don't need an aria-label. we don't need an href for the button. You can also hide the icon from screen readers with aria-hidden.

Reply:
done